contracts/truefi2/SAFU.sol

```diff
@@ -223,10 +223,6 @@ contract SAFU is ISAFU, UpgradeableClaimable {
     }
 
     function _poolLiquidate(ITrueFiPool2 pool, IDebtToken debt) internal {
-        // For legacy LoanTokens:
-        if (loanFactory.isLoanToken(ILoanToken2(address(debt)))) {
-            pool.liquidateLoan(debt);
-        }
         if (loanFactory.isDebtToken(debt)) {
```

Comment:
`require(loanFactory.isDebtToken(debt), "..."`
instead of if?

Reply:
I don't think we need this function anymore, just call `pool.liquidateDebt` from `liquidate`